README.md

```diff
@@ -530,7 +542,7 @@ on the Kubernetes version.
 - **Label the cluster nodes that provide persistent memory device(s)**
 
 ```sh
-    $ kubectl label node <your node> storage=pmem
```

Comment:
What kind of whitespace change is this?

Reply:
'whitespace' may not be the best term here. There is something that does not show properly and is not clean char. In emacs/X I see underscore. In emacs/text I see violet-color block. In vi I see nothing i.e. whitespace.
When I look what bytes really are there, there are actually two bytes 0xc2 and 0xa0.
These somehow appeared already some time ago, was it with version from Mary's editor? (just a guess). Since then, we have gradually replaced some of those. Now my editor saved the contents by overwriting last 4 instances with space.

Reply:
C2A0 has something to do with unicode whitespace.
https://github.github.com/gfm/#unicode-whitespace
But nevertheless, we should not have it I think

Reply:
Yes, I fully agree, I just wondered what it was.